ert_shared/storage/model.py

```diff
@@ -252,3 +252,17 @@ class ErtBlob(Blobs):
 
     def __repr__(self):
         return "<Value(id='{}', data='{}')>".format(self.id, self.data)
+
+
+
+prior_ensemble_association_table = Table('prior_ensemble_association_table', Entities.metadata,
+    Column('prior_id', String, ForeignKey('parameter_priors.key')),
+    Column('ensemble_id', Integer, ForeignKey('ensembles.id'))
+)
+
+class ParameterPrior(Entities):
+    __tablename__ = "parameter_priors"
+
+    key = Column('key', String, primary_key=True)
+    data = Column('data', JSON)
```

Comment:
This kills reverse indexing. Do you foresee that as a problem?

Reply:
i usually not want my primary keys exposed, but that is my opinion..

Reply:
@jondequinor not sure if we need that. Did it the simplest way. If we are to make more of it I think it should be discussed in the squad. 
@oysteoh what do you mean by primary keys?

Reply:
you have added key as a primary key. i don't want to expose my primary keys from the database out in the public. usually i also don't want strings as primary keys - i prefer incremental int's.